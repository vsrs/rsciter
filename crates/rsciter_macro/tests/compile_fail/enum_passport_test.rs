#[derive(rsciter::Passport)]
enum E {}

fn main() {}