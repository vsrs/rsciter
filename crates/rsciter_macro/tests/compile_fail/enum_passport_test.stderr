error: the #[derive(rsciter::Passport)] can only be applied to a struct!
 --> tests/compile_fail/enum_passport_test.rs:2:1
  |
2 | enum E {}
  | ^^^^
